proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -112,18 +112,23 @@ message Validator {
       [(gogoproto.nullable) = false, (gogoproto.stdtime) = true, (gogoproto.moretags) = "yaml:\"unbonding_time\""];
   // commission defines the commission parameters.
   Commission commission = 10 [(gogoproto.nullable) = false];
-  // min_self_delegation is the validator's self declared minimum self delegation.
-  string min_self_delegation = 11 [
-    (gogoproto.moretags)   = "yaml:\"min_self_delegation\"",
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.nullable)   = false
-  ];
-
   // strictly positive if this validator's unbonding has been stopped by external modules
   int64 unbonding_on_hold_ref_count = 12;
-
   // list of unbonding ids, each uniquely identifing an unbonding of this validator
   repeated uint64 unbonding_ids = 13;
+  // Number of shares self bonded from the validator
+  string total_validator_bond_shares = 14 [
```

Comment:
fwiw, anybody can submit a validator bond delegation (this comment should be fixed). 

I'm still on board with the name change, but just checking that doesn't change your opinion

Reply:
The idea of this type of delegation is that it cannot be tokenized and it cannot come from a liquid staker, right? Maybe the name should reflect that. Maybe something like non_liquid_shares.

Reply:
Or even better `liquid_staking_exempt_shares` based on ADR 001. This would also require renaming the `ValidatorBond` message to something like `LiquidStakingExempt`.

Reply:
> The idea of this type of delegation is that it cannot be tokenized and it cannot come from a liquid staker, right?

That is true, but I think this is better defined as a type of delegation that raises the cap of how much can be liquid staked (i.e. cap = validator bond * validator bond factor). 

I think there was a consensus that the original "exempt" naming in ADR 001 was a bit confusing. 

My vote would be `SelfBondShares` as you originally suggested, as this is meant to be a "skin-in-the-game" delegation type.

Reply:
@mpoke friendly bump on this now that you've read through the full ADR. Imo we should either change it to `SelfBondShares` or leave it as `ValdiatorBondShares` - I don't have a strong opinion either way

Reply:
Neither do I. I guess it's okay to leave it as it is. Not the best name, but as long as it's properly documented, it should be fine.